install.php

```diff
@@ -11,7 +11,7 @@
         'sync_backend' => true,
         'rename' => true,
         'prefix' => false,
-        'umlauts' => true,
```

Comment:
Ggf. Ne warning ins system:log schreiben wenn dieses addon via installer aktualisiert wird? (Oder ne info an den user direkt ausgeben?)

Reply:
Für's erste reicht mir das so, bin da gerade zu faul :D. Ich plane nicht in absehbarer Zeit ein Major-Release zu machen, somit wird die Einstellung vermutlich auch noch recht lange drin bleiben.
Wenn es dann wirklich rausfällt, kann man übrigens auch das Update verhindern, wenn die Option gesetzt ist.